test/extended/util/test.go

```diff
@@ -431,6 +435,41 @@ var (
 
 			`Should be able to support the 1\.7 Sample API Server using the current Aggregator`, // down apiservices break other clients today https://bugzilla.redhat.com/show_bug.cgi?id=1623195
 		},
+		"[Skipped:azure]": {
+			"Networking should provide Internet connection for containers", // Azure does not allow ICMP traffic to internet.
```

Comment:
is @openshift/sig-networking aware of this test being disabled for azure?

Reply:
upstream PR for skipping azure https://github.com/kubernetes/kubernetes/pull/79243
^^-- was closed on favor of dropping ping with looks like `nc` based https://github.com/kubernetes/kubernetes/pull/77794#issuecomment-505586864